### DISCUSSION TITLE
How long can I use GitHub Copilot?

### DISCUSSION THREAD

**Discussion by rezaikbal** [community/community#129927] [GitHub Education]
### Hi everyone,

If I use GitHub Copilot, will it expire after I have used it for 1 year and cannot be used again?  Considering I'm still studying for another 2 years, can I apply again?  or how?  Thank You.

**Comment by ryanhaticus** [comment]
having this issue now as well

**Reply by rezaikbal** [reply]
Since when were you approved by GitHub Education? Or have you been using GitHub Copilot for more than 1 year? because I was just approved yesterday as a github edu recipient.
